package.json

```diff
@@ -28,7 +28,7 @@
 		"@types/jest": "^29.5.6",
 		"@typescript-eslint/eslint-plugin": "^6.9.0",
 		"@typescript-eslint/parser": "^6.9.0",
-		"bc-stubs": "github:bananarama92/BC-stubs#v102.0.0",
+		"bc-stubs": "github:bananarama92/BC-stubs#v104.0.0",
```

Comment:
As a heads up: [v104.0.1](https://github.com/bananarama92/BC-stubs/releases/tag/v104.0.1) is now also live with the `BackgroundsTextGet()` type improvements.

Reply:
Yeah, I've used this one instead, thanks!